backend/spec/features/admin/users_spec.rb

```diff
@@ -57,19 +57,20 @@
   shared_examples_for 'a sortable attribute' do
     before { click_link sort_link }
 
-    it "can sort asc" do
+    it "can sort asc", :js do
```

Comment:
what about this `, :js`? Why is this needed?

Reply:
Not sure why it happens, but when using  `:rack_test` driver `expect(page).to have_selector '.sort_link.desc'` fails. It always finds `sort_link.asc` only.

These classes don't seem to be changed by javascript, as clicking the link generates a page reload, so I can't explain why:

```ruby
Capybara.current_driver = :selenium_chrome
=> :selenium_chrome
visit '/admin/users?q[s]=created_at+desc'
=> nil
expect(page).to have_selector '.sort_link.desc'
=> true
Capybara.current_driver = :rack_test
=> :rack_test
visit '/admin/users?q[s]=created_at+desc'
=> nil
expect(page).to have_selector '.sort_link.desc'
RSpec::Expectations::ExpectationNotMetError: expected to find visible css ".sort_link.desc" within #<Capybara::Node::Element tag="table" path="/html/body/div[5]/div/div/table"> but there were no matches
```